src/Xiaomi_Scale.py

```diff
@@ -25,15 +25,15 @@
 import subprocess
 from bluepy import btle
 from bluepy.btle import Scanner, BTLEDisconnectError, BTLEManagementError, DefaultDelegate
-import paho.mqtt.client as mqtt
+import paho.mqtt.publish as publish
 from datetime import datetime
 
 import Xiaomi_Scale_Body_Metrics
 
 # Configuraiton...
 MISCALE_MAC = os.getenv('MISCALE_MAC', '')
-MQTT_USERNAME = os.getenv('MQTT_USERNAME', '')
-MQTT_PASSWORD = os.getenv('MQTT_PASSWORD', '')
+MQTT_USERNAME = os.getenv('MQTT_USERNAME', 'username')
```

Comment:
Why replace `MQTT_USERNAME = os.getenv('MQTT_USERNAME', '')` and `MQTT_PASSWORD = os.getenv('MQTT_PASSWORD', '')` default values?

Reply:
According to that: https://github.com/eclipse/paho.mqtt.python/blob/master/src/paho/mqtt/publish.py#L105
I did it to avoid an if before to set Auth to none or to the wanted dict.
So to send the dict even without Auth, I need a password=none and a username=anything

I find it more readable like that, but I can change if you want me to

Reply:
Thanks 👍